openshift-container-platform-4/policies/CM-Configuration_Management/component.yaml

```diff
@@ -32,9 +32,31 @@
   implementation_status: planned
   narrative:
     - text: |
-        A complete control response is planned. Engineering progress can be
-        tracked via:
+        Configuration management is enforced through Ansible playbooks,
+        Ignition configurations and Operators.
```

Comment:
@vlewisRH This is a great write up! Well done. Question: it seems like Ansible is being deprecated completely in favor of operators. Should we still mention Ansible here?

Reply:
Thank you for your review, Ill make the recommended changes.